tools/convert_feat_to_wholegraph.py

```diff
@@ -90,30 +108,67 @@ def main(folder, feat_names):
                 wgth.destroy_wholememory_tensor(wg_tensor)
 
             # Delete processed feature from memory
-            for t in node_feats_data:
+            for t in feats_data:
                 del t[feat]
+    return feats_data
+
+
+def main(folder, node_feat_names, edge_feat_names):
+    """ Convert node features from distDGL format to WholeGraph format"""
+    os.environ['MASTER_ADDR'] = '127.0.0.1'
+    os.environ['MASTER_PORT'] = '1234'
+    torch.distributed.init_process_group('gloo', world_size=1, rank=0)
+    local_comm = wgth.comm.get_local_device_communicator()
+    wg_folder = os.path.join(folder, 'wholegraph')
+    if not os.path.exists(wg_folder):
+        os.makedirs(wg_folder)
+
+    logging.info("node features:", node_feat_names, " and edge features: ", edge_feat_names, " will be converted to WholeGraph format.")
 
+    metadata = {}
+    # Process node features
+    if node_feat_names:
+        fname_dict = process_node_data(folder, node_feat_names)
```

Comment:
the function name process_node_data is misleading, it is more like get_node_feat_info.